### PR TITLE
ci: fix ci after repo move to bitboxswiss

### PR DESCRIPTION
**Pull request by thisconnect**
The repo changed to github.com/BitBoxSwiss

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -103,7 +103,7 @@ jobs:
           OS_NAME: osx
       - name: Archive app
         run: >
-          pushd ~/go/src/github.com/digitalbitbox/bitbox-wallet-app/frontends/qt/build/osx;
+          pushd ~/go/src/github.com/bitboxswiss/bitbox-wallet-app/frontends/qt/build/osx;
```
Comment (benma) [inline .github/workflows/ci.yml:106]: fyi it's BitBoxSwiss (even though both work), I prefer using the capitalized one for consistency.

Comment (thisconnect) [reply .github/workflows/ci.yml]: done, I used lowercase as most other stuff in go directory is lowercase, but Microsoft :P

**Comment by thisconnect** [comment]
I believe this is unrelated, and a random fail that sometimes occurs.. I'll run pipeline one more time.

```
 === RUN   TestAccountsByKeystore
    accounts_test.go:1435: 
        	Error Trace:	/opt/go/src/github.com/BitBoxSwiss/bitbox-wallet-app/backend/accounts_test.go:1435
        	Error:      	"[0xc0005b3080 0xc0005b4100 0xc0005b3600 0xc0005b4680 0xc0005b3b80]" should have 3 item(s), but has 5
        	Test:       	TestAccountsByKeystore
--- FAIL: TestAccountsByKeystore (0.13s)
```

**Comment by thisconnect** [comment]
@benma ci passed, please let me know if I should just squash all commits or if there is anything else that needs to be changed.

**Review by benma — CHANGES_REQUESTED** [review]
All looks good, but you missed appveyor.yml, which has some `digitalbitbox` folders left. Thanks!

And yes please squash in the end, it's one logical change to move the org.

**Comment by thisconnect** [comment]
I think appveyor ci isn't running at all, @benma could you login and see if we need to activate appveyor for bitboxswiss ?

**Comment by benma** [comment]
> I think appveyor ci isn't running at all, @benma could you login and see if we need to activate appveyor for bitboxswiss ?

I changed the repo url in appveyor, can you push again to see if it will be triggered? Amending commit with no change or a small commit msg change.

**Review by benma — APPROVED** [review]
nice, thanks!!

**Comment by thisconnect** [comment]
ci passed with appveyor, squashing now....

**Pull request merged**